synapse/tests/test_cortex.py

```diff
@@ -1325,3 +1325,8 @@ def test_cortex_int_indx(self):
             self.len(0, core.eval('testint<10'))
             self.len(0, core.eval('testint<20'))
             self.len(1, core.eval('testint<30'))
+
+            list(core.eval('[testint=99999]'))
```

Comment:
what's this for?

Reply:
Imagine a different backing store might implement open vs. closed interval differently.